comp/dogstatsd/packets/types.go

```diff
@@ -36,3 +38,20 @@ type Packets []*Packet
 
 // NoOrigin is returned if origin detection is off or failed.
 const NoOrigin = ""
+
+// SizeOfPacket is the size of a packet structure in bytes
+const SizeOfPacket = unsafe.Sizeof(Packet{})
+
+// SizeInBytes returns the size of the packet in bytes
+func (p *Packet) SizeInBytes() int {
```

Comment:
Would it make sense to call it DataSizeInBytes, to follow the same pattern as the rest of the PR?

Reply:
good idea